mentors/src/data_pipeline/ibm_transcript_service.py

```diff
@@ -0,0 +1,32 @@
+from watson_developer_cloud import SpeechToTextV1
+
+
+"""
+API credentials for the IBM Watson service are given below
+Web login credentials for https://console.ng.bluemix.net/
+Credentials File: Ask Ben Nye for it
+Place the text file containing the credentials file in the root of the website version code
+Contact Madhusudhan Krishnamachari at madhusudhank@icloud.com to get assistance on how to use Watson. (IBM Documentation should help)
+"""
+with open("src/data_pipeline/secrets/credentials.txt", "r") as credentials:
```

Comment:
I wonder about putting secrets under source...could it be closer to the root, under `mentors/secrets` instead?